src/test/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/integration/SoftDeleteJobTest.kt

```diff
@@ -72,15 +71,6 @@ class SoftDeleteJobTest : SqsIntegrationTestBase() {
     assertThat(versions[11]!!.toLocalDate()).isEqualTo(LocalDate.now())
   }
 
-  @Test
-  fun `Unauthorized (401) when no token is supplied`() {
-    webTestClient.get()
-      .uri("/jobs/populate-deleted-at-for-licences/3")
-      .accept(MediaType.APPLICATION_JSON)
-      .exchange()
-      .expectStatus().isEqualTo(HttpStatus.UNAUTHORIZED.value())
-  }
```

Comment:
this was failing previously - no need for this test as no role required and protected by ingress